src/doc/trpl/primitive-types.md

```diff
@@ -1,3 +1,283 @@
 % Primitive Types
 
-Coming Soon!
+The Rust language has a number of types that are considered ‘primitive’. This
+means that they’re built-in to the language. Rust is structured in such a way
+that the standard library also provides a number of useful types built on top
+of these ones, as well, but these are the most primitive.
+
+
+# Unit, or `()`
```

Comment:
Perhaps this could be a subsection of the tuples section? I think it's still pretty commonly referred to as "unit", but putting it by itself may be giving it too much prominence.